### PR TITLE
RAS-1238 Update to docker compose from docker-compose

### PR DESCRIPTION
**Pull request by arroyoAle**
# What and why?
Updating references to the now removed `docker-compose` in favour of `docker compose`
# How to test?
Make sure the tests run properly
# Jira
[RAS-1238](https://jira.ons.gov.uk/browse/RAS-1238)

### CODE REVIEW

**Review by anwilkie — COMMENTED** [review]
This PR is fine, but you won't be able to merge this in until the signed commits are implemented for this repo, which I've done it as part of [this PR](https://github.com/ONSdigital/ras-collection-instrument/pull/335).

File: Pipfile.lock
Comment (matthew-robinson-ons) [inline Pipfile.lock:1]: Lots of changes in here, was it meant to be checked in?

**Review by matthew-robinson-ons — CHANGES_REQUESTED** [review]
Quite a lot of changes to the pipfile.lock are they all needed?

File: Pipfile.lock
Comment (anwilkie) [reply Pipfile.lock:1]: The changes to the dockerfile are similar to the changes I made in my recent PR, so it should be okay.

Comment (arroyoAle) [reply Pipfile.lock:1]: I've updated it to specifically target the ones with vulnerabilities

**Comment by arroyoAle** [comment]
> This PR is fine, but you won't be able to merge this in until the signed commits are implemented for this repo, which I've done it as part of [this PR](https://github.com/ONSdigital/ras-collection-instrument/pull/335).

I've added the signing step

**Review by matthew-robinson-ons — APPROVED** [review]

**Comment by arroyoAle** [comment]
recreated on https://github.com/ONSdigital/ras-collection-instrument/pull/336 with bot signing

**Pull request closed without merging**